tests/IBFE/instrument_panel_01.cpp

```diff
@@ -463,20 +467,19 @@ main(int argc, char** argv)
                     // at data_time = t the bottom face is at z = t and the top face is at z = 1 + t
                     std::unique_ptr<Elem> side_ptr = elem.side_ptr(side_n);
 
-                    double z;
 #if LIBMESH_VERSION_LESS_THAN(1, 7, 0)
-                    z = side_ptr->centroid()(2);
+                    const double z = side_ptr->centroid()(2);
 #else
-                    z = side_ptr->vertex_average()(2);
+                    const double z = side_ptr->vertex_average()(2);
 #endif
-                    if (abs(z - (data_time + 0.0)) < 1.0e-5)
+                    if (std::abs(z - (data_time + 0.0)) < 1.0e-5)
```

Comment:
`abs()` is a C function defined for integers only - I prefer to always write `std::abs()` to make sure we are talking about the overloaded version in C++.